app/controllers/assessments_controller.rb

```diff
@@ -333,55 +333,44 @@ def getAssessmentVariable(key)
 
   # export - export an assessment by saving its persistent
   # properties in a yaml properties file.
+  action_auth_level :exportOptions, :instructor
+  def exportOptions
+
+
+  end
+
   action_auth_level :export, :instructor
   def export
-    base_path = Rails.root.join("courses", @course.name).to_s
-    asmt_dir = @assessment.name
-    begin
-      # Update the assessment config YAML file.
-      @assessment.dump_yaml
```

Comment:
Where did this end up in the final export?

Reply:
this is part of the dump everything we dont use the yaml files so i didnt see a need to just export them all by them selfs